apps/shooting-clubs/translations/src/en/pages.json

```diff
@@ -10,7 +10,7 @@
   },
   "second-contact-name": {
     "header": "Who should we contact in {{values.club-secretary-name}}'s absence?",
```

Comment:
cool, I could have just added triple braces for all situations but since it wasn't coming back with any issues, I left it